src/examples/graph.js

```diff
@@ -54,7 +54,7 @@ const sample: IGraph = {
     {
       handleText: '5',
       handleTooltipText: '5',
-      source: 'start1',
+      source: 0,
```

Comment:
You can create a test for this, but please don't modify the examples.

Reply:
Ok. Added some test cases and fixed the change.